### DISCUSSION TITLE
Settings/Steps

### DISCUSSION THREAD

**Discussion by Deph1** [volotat/SD-CN-Animation#48] [General]
Hello!

Could you please share your settings/steps taken to create vid2vid or txt2vid, I find that mine are not doing so well even with using Hed and softhed as the preprocessor. I tend to find it's just "illustrating" closely to the original.

**Comment by volotat** [comment]
Hello, I have plans to make a video guide for the extension, showing the set-up I use and explaining the general idea of how vid2vid and txt2vid algorithms work. But I think I will do it after the major problem with the extension is fixed, so do not expect to see it too soon.

But for a quick answer here are my typical settings I use to test the extension
![image](https://github.com/volotat/SD-CN-Animation/assets/14307880/b8cb98e3-4117-4e4b-86a9-c5cf8a2b9687)

And here is the kind of result I'm getting with it
https://streamable.com/vf7x0u
